mmv1/products/discoveryengine/DataConnector.yaml

```diff
@@ -0,0 +1,331 @@
+update_verb: 'PATCH'
+update_mask: true
+delete_url: 'projects/{{project}}/locations/{{location}}/collections/{{collection_id_url_param}}'
+import_format:
+  - 'projects/{{project}}/locations/{{location}}/collections/{{collection_id}}/dataConnector'
+timeouts:
+  insert_minutes: 20
+  update_minutes: 5
+  delete_minutes: 20
+autogen_async: false
+async:
+  actions: ['create', 'delete']
+  type: 'OpAsync'
+  operation:
+    base_url: '{{op_id}}'
+  result:
+    resource_inside_response: true
+custom_code:
+  post_read: 'templates/terraform/post_read/discoveryengine_dataconnector_use_declared_res.go.tmpl'
+examples:
+  - name: 'discoveryengine_dataconnector_jira_basic'
+    primary_resource_id: 'jira-basic'
+    vars:
+      collection_id: 'collection-id'
+      client_id: 'client-id'
+      client_secret: 'client-secret'
+    test_vars_overrides:
+      client_id: '"tf-test-client-id"'
+      client_secret: '"tf-test-client-secret"'
+parameters:
+  - name: 'location'
+    type: String
+    description: |
+      The geographic location where the data store should reside. The value can
+      only be one of "global", "us" and "eu".
+    url_param_only: true
+    required: true
+    immutable: true
+  - name: 'collectionIdUrlParam'
+    type: String
+    description: |
+      The ID to use for the Collection, which will become the final component
+      of the Collection's resource name. A new Collection is created as
+      part of the DataConnector setup. DataConnector is a singleton
+      resource under Collection, managing all DataStores of the Collection.
+      This field must conform to [RFC-1034](https://tools.ietf.org/html/rfc1034)
+      standard with a length limit of 63 characters. Otherwise, an
+      INVALID_ARGUMENT error is returned.
+    url_param_only: true
+    required: true
+properties:
+  - name: 'collectionId'
+    type: String
+    description: |
+      The ID to use for the Collection, which will become the final component
+      of the Collection's resource name. A new Collection is created as
+      part of the DataConnector setup. DataConnector is a singleton
+      resource under Collection, managing all DataStores of the Collection.
+      This field must conform to [RFC-1034](https://tools.ietf.org/html/rfc1034)
+      standard with a length limit of 63 characters. Otherwise, an
+      INVALID_ARGUMENT error is returned.
+    required: true
+    ignore_read: true
+  - name: 'collectionDisplayName'
+    type: String
+    description: |
+      The display name of the Collection.
+      Should be human readable, used to display collections in the Console
+      Dashboard. UTF-8 encoded string with limit of 1024 characters.
+    required: true
+    ignore_read: true
+  - name: 'dataConnector'
+    type: NestedObject
+    description: |
+      The DataConnector to initialize in the newly created Collection.
+    required: true
+    ignore_read: true
```

Comment:
I'm a little confused to see this field marked as `ignore_read: true`, but then many of the fields within in are `output: true`. Could you clarify the intent?